packages/components/src/components/info-card.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+import { cn } from '../cn';
+import { Stud } from './stud';
+
+export interface InfoCardProps extends React.HTMLAttributes<HTMLElement> {
+  icon: ReactNode;
+  heading: ReactNode;
+  as?: 'div' | 'li';
+}
+
+export function InfoCard({
+  as: Root = 'div',
+  icon,
+  heading,
+  className,
+  children,
+  ...rest
+}: InfoCardProps) {
+  return (
+    <Root className={cn('bg-beige-100 p-6 md:p-12', className)} {...rest}>
+      <Stud>{icon}</Stud>
+      <h3 className="mt-4 text-xl font-medium leading-[1.4] text-green-1000 md:mt-6">{heading}</h3>
```

Comment:
```suggestion
      <h3 className="mt-4 text-xl font-medium leading-[1.4] text-green-1000 md:mt-6">{heading}</h3>
```
what's 1.4 stay for I guess it's 18px?

Reply:
It's not `18px` because it's `text-xl`. Just 1.4 of font-size. Copied it from Figma.

We can prolly remove it, as it's `20px * 1.4 == 28px`, and Tailwind's `text-xl` includes `line-height: 1.75rem` which, with 16px html font-size is also `28px`.

<img width="825" alt="image" src="https://github.com/user-attachments/assets/6f4b2715-bd11-4934-83e9-f0edbf02eaea">

Reply:
oh thx, and yes you right, you can remove it, also for the future there is shorthand `text-xl/[value-of-your-leading]`